libarchive/test/test_write_format_zip_entry_size_unset.c

```diff
@@ -107,10 +107,22 @@ static void write_archive(struct archive *a)
 
 static void verify_contents(const char *zip_buff, size_t size)
 {
-	unsigned long crc = bitcrc32(0, file_data1, sizeof(file_data1));
-	crc = bitcrc32(crc, file_data2, sizeof(file_data2));
+	const unsigned long crc = bitcrc32(
+			bitcrc32(0, file_data1, sizeof(file_data1)), file_data2, sizeof(file_data2)
+	);
```

Comment:
Just use two variables `crc_foo and `crc`. The code looks quite uncommon and "ugly" IMHO